lms/djangoapps/email_marketing/models.py

```diff
@@ -0,0 +1,57 @@
+"""
+Email-marketing-related models.
+"""
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from config_models.models import ConfigurationModel
+
+
+class EmailMarketingConfiguration(ConfigurationModel):
+    """ Email marketing configuration """
+
+    class Meta(object):
+        app_label = "email_marketing"
+
+    sailthru_enabled = models.BooleanField(
+        default=False,
+        help_text=_('Enable Sailthru email marketing support.')
+    )
```

Comment:
`ConfigurationModel` comes with an `enabled` field. Do you need a separate field for SailThru specific enablement?